src/Raven.Client/Documents/Session/IRevisionsSessionOperationsAsync.cs

```diff
@@ -65,5 +72,16 @@ public interface IRevisionsSessionOperationsAsync
         /// Returns the number of revisions for specified document.
         /// </summary>
         Task<long> GetCountForAsync(string id, CancellationToken token = default);
+
+    }
+
+    public interface ILazyRevisionsOperationsAsync
+    {
+        Lazy<Task<List<T>>> GetForAsync<T>(string id, int start = 0, int pageSize = 25, CancellationToken token = default);
```

Comment:
Can you copy the comments from 'IRevisionsSessionOperationsAsync' ?

Reply:
done

Reply:
Ping

Reply:
done